python_ta/__init__.py

```diff
@@ -212,7 +212,12 @@ def reset_linter(config=None, file_linted=None):
          {'default': 'http://127.0.0.1:5000',
           'type': 'string',
           'metavar': '<server-url>',
-          'help': 'Server address to submit anonymous data'})
+          'help': 'Server address to submit anonymous data'}),
+        ('pycodestyle-ignored-tests',
```

Comment:
This is the right option to add, but you should add it directly to the checker, not here. Some of the other custom checkers do this, if you want to take a look.